test/lib/rails-jquery-autocomplete_test.rb

```diff
@@ -15,22 +15,22 @@ class ::Movie ; end
       end
 
       should 'respond to the action' do
-        assert_respond_to @controller, :autocomplete_movie_name
+        assert_respond_to @controller, :autocomplete_action_name
       end
 
       should 'render the JSON items' do
-        mock(@controller).get_autocomplete_items({
-          :model => Movie, :method => :name, :options => @options, :term => "query"
-        }) { @items }
+        mock(@controller).get_autocomplete_items(
+            { :model => Movie, :options => @options, :term => "query", :method => :name }
+        ) { @items }
 
         mock(@controller).json_for_autocomplete(@items, :name, nil)
-        get :autocomplete_movie_name, :term => 'query'
+        get :autocomplete_action_name, :term => 'query'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.